go/pkg/bertymessenger/handlers.go

```diff
@@ -63,27 +133,26 @@ func accountServiceTokenAdded(svc *service, gme *bertytypes.GroupMetadataEvent)
 		return errcode.ErrDeserialization.Wrap(err)
 	}
 
-	if err := addServiceToken(svc.db, bytesToString(config.AccountPK), ev.ServiceToken); err != nil {
+	if err := h.db.addServiceToken(bytesToString(config.AccountPK), ev.ServiceToken); err != nil {
```

Comment:
I know you didn't introduced this `bytesToString` helper, but since this PR is about refactoring, I think we should rename it to something more related to what it actually does